mvlearn/model_selection/split.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Authors: Alexander Chang, Gavin Mischler
+
+import numpy as np
+from sklearn import model_selection as ms
+
+
+def train_test_split(*inputs, test_size=None, train_size=None,
```

Comment:
```suggestion
def train_test_split(*inputs, *, test_size=None, train_size=None,
```
to force passing param names

Reply:
The build failed when I changed to this, saying Invalid Syntax. A quick test in a jupyter notebook makes me think Python doesn't allow you to use * after your first argument is *inputs rather than inputs. So, I changed to **options for all the kwargs to follow sklearn's API completely